clients/clients.yaml

```
@@ -0,0 +1,16 @@
+ndoli:
+  tests:
+    git-annex-home: |
+      set -ex
```

may be even 
```suggestion
      set -exu
```
since variables are used

---

Done.